docs/getting-started/codacy-quickstart.md

```diff
@@ -49,23 +49,8 @@ Click the link **Go to repository** to see the [code quality overview of your re
 
 ![Repository dashboard](../repositories/images/repository-dashboard.png)
 
-## 4. Tweak your repository settings {: id="tweak-settings"}
+**Congratulations, your new repository is ready!** To explore the initial analysis results, [check the **Issues** page](../repositories/issues.md).
 
- Check the static analysis results on the **Issues** page to [explore the initial analysis results](../repositories/issues.md).
+## Next steps
 
- The first analysis is based on Codacy **default configurations**, so you should tweak and adapt the following configurations to match your preferences and specific scenario:
-
--   [Ignore files](../repositories-configure/ignoring-files.md) that you want to exclude from the Codacy analysis
--   [Configure the code patterns](../repositories-configure/configuring-code-patterns.md) that Codacy uses to analyze your repository
--   [Adjust the quality settings](../repositories-configure/adjusting-quality-settings.md) to control quality gate that Codacy applies to your commits and pull requests
-
-## You're all set! 🎉 {: id="all-set"}
-
-Congratulations, your new repository is ready!
-
-Optionally, you can also:
-
--   [Add coverage reports to Codacy](../coverage-reporter/index.md)
--   Configure Codacy to provide analysis feedback directly on [GitHub](../repositories-configure/integrations/github-integration.md#configuring), [GitLab](../repositories-configure/integrations/gitlab-integration.md#configuring), or [Bitbucket](../repositories-configure/integrations/bitbucket-integration.md#configuring) pull requests
--   [Use Codacy as a quality gate](../faq/general/how-do-i-block-merging-prs-using-codacy-as-a-quality-gate.md) to block merging pull requests that don't meet your quality standards
--   [Add a Codacy badge to your repository](adding-a-codacy-badge.md) displaying the current code quality grade or code coverage
+The first analysis is based on default tool and pattern configurations. [It's important that you configure your repository](configuring-your-repository.md) to integrate code analysis seamlessly into your existing pipeline.
```

Comment:
A small suggestion to allow the reader to read all the sentence before they get distracted by the link.

```suggestion
The first analysis is based on default tool and pattern configurations. It's now important that you configure your repository to integrate code analysis seamlessly into your existing pipeline. See how to [configure your repository to match the use cases of your team](configuring-your-repository.md).
```